samples/media/video/src/main/java/com/example/platform/media/video/TransformerVideoComposition.kt

```diff
@@ -157,14 +161,27 @@ class TransformerVideoComposition : Fragment() {
     private fun createComposition(): Composition {
         val video1 = EditedMediaItem.Builder(
             // apply effects only on the first item
-            MediaItem.fromUri(URI_ITEM1))
+            MediaItem.fromUri(URI_ITEM1),
+        )
             .setEffects(getSelectedEffects())
             .build()
         val video2 = EditedMediaItem.Builder(
-            MediaItem.fromUri(URI_ITEM2))
+            MediaItem.fromUri(URI_ITEM2),
+        )
+            .build()
+        val image = EditedMediaItem.Builder(
+            MediaItem.Builder().setUri(URI_IMAGE_ITEM).build(),
+        )
+            .setDurationUs(3_000_000)
+            .setFrameRate(30)
             .build()
         val compositionSequences = ArrayList<EditedMediaItemSequence>()
-        val videoSequence = EditedMediaItemSequence(ImmutableList.of(video1, video2))
+
+        // This works
+        val videoSequence = EditedMediaItemSequence(ImmutableList.of(video1, image, video2))
+        // This doesn't work and results in a 0 second video of the image being exported
+//        val videoSequence = EditedMediaItemSequence(ImmutableList.of(image, video1, video2))
```

Comment:
@droid-girl I attempted to add an image before the video here, and this didn't work and a video of 0 seconds with just the image was exported.

However, if the image is added in between the two videos (ie: line 181), everything seems to work fine.

Any idea what I'm doing wrong here?